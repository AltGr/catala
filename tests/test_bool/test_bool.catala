/*
declaration scope TestBool :
	context foo content bool 
	context bar content int

scope TestBool :
  def bar = 1
  def foo ? bar >= 0 ~= true
	def foo ? bar < 0 ~= false 
*/
