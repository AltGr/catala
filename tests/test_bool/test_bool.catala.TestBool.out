[RESULT] bar -> 1
[RESULT] foo -> true
